pkg/apis/common/gs/resource_test.go

```diff
@@ -0,0 +1,25 @@
+package gs
+
+import (
+	"encoding/json"
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+func TestCommonGenericServiceAPIResources(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "test suite for common gs API resources")
+}
+
+var _ = DescribeTable("PartialResourceList", func(list PartialResourceList, expected string) {
```

Comment:
when you move the func to the next line, the block looks a bit less weird, with the Entries following it